ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile

```diff
@@ -4,22 +4,23 @@
 
 FROM registry.redhat.io/codeready-workspaces/plugin-java11-rhel8:latest
 
-ENV MANDREL_VERSION=20.1.0.3.Final
-ENV QUARKUS_VERSION=1.7.6.Final-redhat-00014
-ENV KN_VERSION=0.17.3
-ENV OC_VERSION=4.6
+ENV MANDREL_VERSION=20.3.1.2-Final
+ENV QUARKUS_VERSION=1.11.6.Final-redhat-00001
+ENV TKN_VERSION=0.15.0
+ENV KN_VERSION=0.19.1
+ENV OC_VERSION=4.7
 ENV GRAALVM_HOME="/usr/local/mandrel-java11-${MANDREL_VERSION}"
 ENV PATH="/usr/local/maven/apache-maven-${MVN_VERSION}/bin:${PATH}"
 
 USER root
 
-RUN wget -O /tmp/oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/${OC_VERSION}/linux/oc.tar.gz && cd /usr/bin && tar -xvzf /tmp/oc.tar.gz && chmod a+x /usr/bin/oc && rm -f /tmp/oc.tar.gz
+RUN wget -O /tmp/oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz && cd /usr/bin && tar -xvzf /tmp/oc.tar.gz && chmod a+x /usr/bin/oc && rm -f /tmp/oc.tar.gz
```

Comment:
I like to say latest == lazy 😁 

Is there a reason you need this to be latest instead of pegged to a version? This is how you get..."unexpected features".

Reply:
There're 2 reasons for this. 1) This dockerfile is only used when we built a new custom CRW image for OCP version upgrade and relevant tools. So this dockerfile won't use for every single provision. 2) When I tried to download the latest version with a version number(i.e. OCP 4.7.2), it didn't work due to the download page didn't update for the new version but the latest is always working even if this docker build runs manually. WDYT? Do you still need me to rollback to version specific stuff?

Reply:
I think you should get the client from here:

https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.7.2/

The oc directory you are trying to pull from doesn't even look like it has gotten new versions since dec-2020, but latest is still recent. Totally sus.

And I really don't think you should use latest for anything. Sure, it won't be used for every provision, but when someone goes and builds it again and 4.8 was just set to latest but you are still doing 4.7, you might be unpleasantly surprised.

Reply:
Fair enough. Just updated PR.

Reply:
Take a look at this. Let me know if you have any questions :)

Reply:
thanks, looks okay.